tests/unit/records/test_records_jsonschemas.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2016 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this licence, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+import json
+import os
+import pkg_resources
+
+from jsonschema import Draft4Validator
+
+
+def fetch_schema(filename):
+    return json.loads(
+        pkg_resources.resource_string(
+            'inspirehep',
+            os.path.join(
+                'modules',
+                'records',
+                'jsonschemas',
+                'records',
+                filename
+            )
+        )
+    )
+
+
+def test_acquisition_source_schema_is_valid():
+    acquisition_source_schema = fetch_schema('acquisition_source.json')
```

Comment:
I'm 👍  for automatically testing all `.json` files in the folder instead of listing them.

I *think* the answer for the `Draft4Validator` is yes:
"resolver – an instance of RefResolver that will be used to resolve $ref properties (JSON references). If unprovided, one will be created." (from http://python-jsonschema.readthedocs.io/en/latest/validate/#the-validator-interface)